.build/build-rat.xml

```diff
@@ -106,7 +106,7 @@
         </fail>
     </target>
 
-    <target name="_assert_rat_output">
+    <target name="_assert_rat_output" unless="${rat.skip}">
```

Comment:
```suggestion
    <target name="_assert_rat_output" unless="rat.skip">
```

?

Reply:
what do you mean? it is defined in the same way as in https://github.com/apache/cassandra/pull/2434/files#diff-d274784b2fecdbee879b6175c05a9860374c65c0c0b7e87efe8ec66c0fc3d838R50